pages/phases/getting-started/identifying-appropriate-positions.md

```diff
@@ -15,13 +15,15 @@ intro: To meet the goals of your hiring action, gather information to drive your
 
 ### Identifying a Classified Position Description
 
-Your position description (PD) should reflect adequate specialized experience to require the time and effort of SMEs. Rather than classifying a new PD, using a standardized PD is encouraged since you will be doing an in-depth job analysis workshop to link the PD to an accurate job announcement. The competencies identified through the job analysis workshop simply need to be rooted in the PD. <a href="https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/deo_handbook.pdf">Page 3-12 of the updated DE Handbook clarifies</a>:
+Your position description (PD) should reflect adequate specialized experience to require the time and effort of SMEs. Rather than classifying a new PD, we encourage using a standardized PD since your in-depth job analysis workshop will link the PD to an accurate job announcement. The competencies identified through the job analysis workshop simply need to be rooted in the PD. <a href="https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/deo_handbook.pdf">Page 3-12 of the updated DE Handbook clarifies</a>:
 
 <blockquote>"Conditions of employment should be supported in the position description and/or by job analysis."</blockquote>
 
 
 ### Identify Billets
 
-To make this type of hiring action effective, you should have approved headcount of five or more GS-12+ billets for the same technical role. Fewer billets may not provide the necessary return on SME time required for this process.
+Our research suggests that the typical cost to hire a new full time employee in industry is $4000-$5000. One pilot agency spent ~$13000 (in personnel costs) on their hiring action, which onboard four people, suggesting a cost per hire of $3250. Other pilots have spent between $2500 and $6900 per hire on their SMEQA pilots.
```

Comment:
i really don't like talking about cost to agency because HR specialists are going to be very confused by this and have no baseline of how much time they spend now per hiring action. So this will leave them scratching their heads at best and leaving thinking this is going to cost them $13k somehow at worst. I love the CBA work we did but think we need to leave it off the website and put it into powerpoint slides to be able to speak to it when other stakesholders (agency leadership) are interested. this is just not the right audience.

here's what i originally wrote as an alternative but don't think this really helps our audience.

"Our research suggests that the typical cost to hire a new full time employee in industry is $4,000-$5,000. One pilot agency's personnel time spent on the pilot would translate to a cost of $3,250 per hire since they onboarded four applicants. This amount fluctuates based on the number of hires: more hires translates to a lower amount and fewer hires translates to a higher amount."

Reply:
Got it; will revise